src/ui/profile_preferences.ui

```diff
@@ -3434,7 +3434,7 @@ you can use it but there could be issues with aligning columns of text</string>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QWidget" name="widget_timerDebugOutputMinimumInterval" native="true">
-            <layout class="QHBoxLayout" name="horizontalLayout_timerDebugOutputMinimumInterval" stretch="0,1">
+            <layout class="QHBoxLayout" name="horizontalLayout_widget_timerDebugOutputMinimumInterval" stretch="0,1">
```

Comment:
56 especially! this is not friendly for humans coding

Reply:
Similarly:
```suggestion
            <layout class="QHBoxLayout" name="horizontalLayout_w_timerDebugOutputMinimumInterval" stretch="0,1">
```
but that does not help so much...

OTOH Layouts are generally (IIRC) only seen within the `.ui` file - and TBH a long and descriptive name that can be related to the parts around it actually makes it easier to keep track of which bit you are looking at.